pkgs/development/python-modules/color-matcher/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  numpy,
+  imageio,
+  docutils,
+  ddt,
+  matplotlib,
+  packaging,
+}:
+let
+  pname = "color-matcher";
+  version = "0.6.0";
+in
+buildPythonPackage {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-e6igB4LD5eWTHdp7H7nFcqzoLeDGyXZUQyt8/gqnSEM=";
+  };
```

Comment:
It would be nice to build from `https://github.com/hahnec/color-matcher` and have the tests suite running